README.md

```diff
@@ -71,21 +85,21 @@ requirements refer to [pv_cheetah.h](/include/pv_cheetah.h). The following trans
 resource directory.
 
 ```bash
-python demo/python/cheetah_demo.py --audio_paths resources/audio_samples/test.wav
+python demo/python/cheetah_demo.py --audio_paths resources/audio_samples/test.wav --license_path PATH_TO_YOUR_CHEETAH_LICENSE_FILE
```

Comment:
We used ${PATH_TO_YOUR_CHEETAH_LICENSE_FILE} notation for variables in shell across documentation.